src/components/dialogs/create-study-dialog.js

```diff
@@ -365,8 +422,50 @@ export const CreateStudyDialog = ({ open, onClose }) => {
                         style={{ width: '90%' }}
                         label={<FormattedMessage id="studyDescription" />}
                     />
-                    {caseExist && <SelectCase />}
-                    {!caseExist && <UploadCase />}
+                    {!providedCase ? (
+                        (caseExist && <SelectCase />) ||
+                        (!caseExist && <UploadCase />)
+                    ) : (
+                        <>
+                            <table>
+                                <tbody>
+                                    <tr>
+                                        <th>
+                                            <Button
+                                                onClick={handleSelectFolder}
+                                                variant="contained"
+                                                style={{
+                                                    paddingLeft: '30px',
+                                                    paddingRight: '30px',
+                                                }}
+                                                color="primary"
+                                                component="label"
+                                            >
+                                                <FormattedMessage id="showSelectDirectoryDialog" />
+                                            </Button>
+                                        </th>
+                                        <th>
+                                            <p>{activeDirectoryName}</p>
```

Comment:
Maybe display the full directory path instead of just the name ?

Reply:
I'll try to implement this